bitnami/wordpress/values.yaml

```diff
@@ -465,3 +465,13 @@ metrics:
 ##         containerPort: 1234
 ##
 sidecars: {}
+
+## Add init containers to the pod.
+## ref: https://kubernetes.io/docs/concepts/workloads/pods/init-containers/
+## Example:
+## initContainers:
+##  - name: update-wp-content
+##    image: busybox:1.28
+##    command: ['sh', '-c', 'copy themes and plugins from git and push to /bitnami/wordpress/wp-conntent. Should work with extraVolumeMounts and extraVolumes']
+##
+initContainers: {}
```

Comment:
You need to apply the same changes in the `values-production.yaml` and document this parameter in the **README.md**